components/Common/ShellBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { LocaleProvider } from '../../../../providers/localeProvider';
+import userEvent from '@testing-library/user-event';
+import { AppProps } from '../../../../types';
+import ShellBox from '../index';
+
+const mockWriteText = jest.fn();
+const i18nData = { currentLocale: { code: 'en' } } as AppProps['i18nData'];
+
+Object.defineProperty(window, 'navigator', {
```

Comment:
Again, I feel like we should assign this on `beforeEach` and restore the original value on `afterEach`